packages/button/src/ButtonContent/ButtonContent.tsx

```diff
@@ -50,6 +50,17 @@ export const ButtonContent = (props: ButtonContentProps) => {
     return unregisterRipple;
   }, [rippleRef, variant, darkMode, disabled, theme]);
 
+  const spinner =
+    loadingIndicator &&
+    React.cloneElement(loadingIndicator, {
+      className: cx({
+        [centeredSpinnerStyles]: !loadingText,
```

Comment:
Should also allow the user to pass their own className to Spinner

Reply:
updated